### ISSUE TITLE
As a user I would like to have correct contact info available in the app 

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#110]
## Description
Give the app owner the possibility to include contact info. This should be presented in a universal way, so that it is presented the same on all apps using the Altinn design. As a first version we propose that we create a footer that the app developer can populate with information.

## Tilgjengelighetserklæring
Kan denne footeren også inneholde lenke til tilgjengelighetserklæring?
## Screenshots
![Skjermbilde 2021-10-25 kl  09 11 32](https://user-images.githubusercontent.com/33222679/138650625-8ad6ab31-2efd-40e7-87d5-f73163ea9029.png)

## Considerations
What kind of contact info do we need to support?
- Mail
- Phone
- Chatbot?

## Specification tasks
- [ ] Should we support tilgjengelighetserklæring in the footer?
- [ ] Create designs for all channels that can be shown in footer
- [ ] Mobile view

**Comment by yne-bv** [comment]
Patentstyret would also like this feature

**Comment by FinnurO** [comment]
#538
